scripts/linz-bde-schema-publish.in

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
+
+DB_NAME=
+export PSQL=psql
+
+if test "$1" = "--version"; then
+    echo "@@VERSION@@ @@REVISION@@"
+    exit 0
+fi
+
+if test -n "${BDESCHEMA_SQLDIR}"; then
+    SCRIPTSDIR=${BDESCHEMA_SQLDIR}
+fi
+
+while test -n "$1"; do
+    DB_NAME=$1; shift
+done
+
+if test -z "$DB_NAME"; then
+    echo "Usage: $0 { <database> | - }" >&2
+    echo "       $0 --version" >&2
+    exit 1
+fi
+
+export PGDATABASE=$DB_NAME
+
+# Configure environment for dbpatch-loader
+PGBIN=`pg_config --bindir 2> /dev/null`
+if test -n "$PGBIN"; then
+    PATH=$PATH:$PGBIN
+else
+    # Wild guess where table_version-loader and dbpatch-loader
+    # will be found (as of versions 1.4.0 and 1.2.0 they were
+    # installed in PostgreSQL bin dir)
+    for f in /usr/lib/postgresql/*/bin/; do
+      PATH="$PATH:$f";
+    done
+fi
```

Comment:
* shouldn't be needed anymore as those scripts are installed in `/usr/bin` now. 
* also 'table_version-loader and dbpatch-loader' looks like copy/paste error